scripts/install_nodejs_14.bash

```diff
@@ -0,0 +1,21 @@
+#
+# Install Node.js 14 on Ubuntu
+#
+# Requirements for development machines
+#
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+IFS=$'\n\t'
+
+sudo apt update
+
+# Script to install the NodeSource Node.js 14.x repo onto a Debian or Ubuntu system
+curl -sL https://deb.nodesource.com/setup_14.x | sudo bash -
```

Comment:
Question: Is this the offical version of node.js that we should support? For Simphony and osrparc-s4l I am using v 12.

Reply:
I put the same as in the the CI (see .github folder), therefore this is our "official" version :-)